backend/emails/accepted/html.pug

```diff
@@ -1,8 +1,9 @@
 // pug template for sending emails after application
 // test online with https://pughtml.com
 p Hi #{name},
-p We are glad to announce you that you will be a members of #{campaignName}!
+p We are glad to announce you that you will be a member of #{campaignName}!
+p Class starts on (insert date & time)! Be there, on time!
```

Comment:
For the first round, write directly the date & time. We could imagine add this date later on the database.

Reply:
So know I should edit the:
Campaign name (I guess it means the class name, e.g. 9th Coding Class o just Coding Class?)
Location (is Athens right?) 

These names, where should I change them? In these texts or in another database?